chain-core/src/main.rs

```diff
@@ -0,0 +1,5 @@
+// this is dummy function to build WASM
+fn main()
+{
+    println!("chain-core wasm ok");
+}
```

Comment:
not need to put this in chain-core -- the readme doc can just have an example crate with main.rs that imports chain-core and does some simple stuff (creates a signed transaction? prints tx id?) and one can compile and run in WASM

Reply:
agree, i'll fix it